dist/js/apps/events-page/app.js

```diff
@@ -1140,6 +1155,11 @@ var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol
 			filterHelperService.clearQueryParams();
 
 			vm.filterEvents(vm.requestModel);
+
+			trackEvent({
+				action: 'Clear All Filters',
+				category: 'Events'
```

Comment:
Since `'events'` is repeated a few times it should be assigned to a `const`.

Reply:
@ascire this has been moved to constants